jvm/ml4ir-inference/src/test/integration/ml4ir/inference/tensorflow/TensorFlowInferenceIT.scala

```diff
@@ -119,7 +119,7 @@ class TensorFlowInferenceIT extends TestData {
       )
     }
     if (query.predictedScores != null) {
-      assertArrayEquals("scores aren't close enough: ", docScores, query.predictedScores, 1e-6f)
+      assertArrayEquals("scores aren't close enough: ", docScores, query.predictedScores, 1e-4f)
```

Comment:
@jakemannix @ducouloa FYI

Reply:
Was failing for this reason
```
org.junit.internal.ArrayComparisonFailure: scores aren't close enough: : arrays first differed at element [0]; expected:<0.63819975> but was:<0.63819844>
```

Reply:
do we understand why? You may want to add an inline comments saying we had 1e-6f but it was too much.